builtin/providers/aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -149,10 +147,6 @@ func testAccCheckESDomainDestroy(s *terraform.State) error {
 const testAccESDomainConfig = `
 resource "aws_elasticsearch_domain" "example" {
   domain_name = "tf-test-1"
-
-  tags {
```

Comment:
Just a FYI, the addition of tags to the basic test actually masked this issue. There is another test in place that checks for tags